docs/adr/0009-naming-conventions.md

```diff
@@ -0,0 +1,158 @@
+# Naming conventions
+
+Date: 2020-09-24
+
+## Status
+
+2020-12-02 Proposed
+
+## Component files
+
+### Principles
+
+- Keep it simple, reduce nesting
+- A `src/pages` folder will hold 'root' components and reflect the navigation structure (menu, routes) of the app
+- Components needed only by one 'root' component go into that root component's folder, no extra nesting
```

Comment:
Agreed if each component has its own folder together with it's tests (accepting the fact that almost each component will have its own test).
```
kto
      Form
          Form.tsx
          Form.test.tsx
          index.ts
      Kto.tsx
      Kto.test.tsx
      index.ts
```

Otherwise the size of a main component folder will be to large and difficult to understand.